src/ticketmap/tooltip.js

```diff
@@ -1,58 +1,71 @@
-import CSSTransition from 'react-transition-group/CSSTransition';
-
-import { mainContainerStyle } from './styles';
+import PropTypes from 'prop-types'
```

Comment:
Why are we adding prop types?

Reply:
`standard-react` requires prop-type definitions on all components

Reply:
We should strive to stick to standard as much as possible. It should make the project easier to read and modify by other developers and people integrating the library into their site. However, I think forgoing the prop types, for the time being, is the right choice (given we're moving to TypeScript in the immediate future)